frappe/desk/doctype/todo/todo.json

```diff
@@ -13,7 +13,7 @@
   "column_break_2",
   "color",
   "date",
-  "owner",
+  "allocated_to",
```

Comment:
What's the reason for this change? Is it related to the CVE report linked?

Reply:
Yeah [here is the link for the CVE report](https://erpnext.com/app/cve-report-form/VUL-281910).
While raising the fix for this issue the test cases were failing and while fixing those test cases I encountered this issue of ToDo doctype having an extra owner field.

Reply:
I read the report prior to the review. I'm asking why have you chosen to fix it this way. What other alternative solutions have you considered, and why was this chosen?

Going down this route, there are other DocTypes with an `owner` field defined in their schema too, defined from their respective JSON notations. Would they require the same changes? Does this not require a patch?